3-get-session-info.md

```diff
@@ -0,0 +1,65 @@
+# Call Kobiton REST API to get session information
+This part will demonstrate how to get session information with Kobiton rest api. 
+>Kobiton have supported multiple language for [API documentation](add_api_doc_url). Go to docs for further language support
+
+## Prerequisites
+  - Kobiton account.
+
+## 1. Authentication
+To make a request:
+- Encode your credentials in base64 for HTTP Basic Authenitication. You can run this command below to get the encoded token:  
+`echo -n <your_username>:<your_api_key>`
+
+
+## 2. Get session information through Kobiton REST API
+- To send an API request:
+  ```
+  curl -X GET https://api.kobiton.com/v1/{request_path}
+  -H 'Authorization: Basic dGVzdHVzZXI6MTIzZWQtMTIzZmFjLTkxMzdkY2E='
+  -H 'Accept: application/json'
+  ```
+
+### 2.1 Get Application Info
+`GET /apps/{application_ID}`  
+You can get your application id in your desiredCaps.
+
+### 2.2 Get Session Info
+`GET /session/{sessionID}`
+
+Response elements:
+- `state`: Test final result
+- `deviceBooked`: Check if the device is booked
+- `log`: The log URL and video URL
+  >Note: Log URL and video URL might take a while to be uploaded to Kobtion Portal. Therefore, you will have to wait for a while before getting your session information.
+
+For more information, visit [Kobiton API Document - Get a session](https://api.kobiton.com/docs/?javascript--nodejs#get-a-session)  
+
+### 2.3 Get Session Commands
+`GET /session/{sessionId}/commands`
+
+To get a certain page of your commands, add `page` parameter in your query
+For example:
```

Comment:
"To get a certain page of your commands, add page parameter in your query For example:" <- this is what it looks like
"To get a certain page of your commands, add page parameter in your query.
For example: " <- this is what you want it to look like, add a new line in between the lines! Or you can add a period after "query".